specification/_types/aggregations/pipeline.ts

```diff
@@ -157,41 +161,14 @@ export class InferenceAggregation extends PipelineAggregationBase {
   inference_config?: InferenceConfigContainer
 }
 
-export class InferenceConfigContainer {
+/** @variants container */
+class InferenceConfigContainer {
```

Comment:
This name is unfortunate as it's in the global namespace, would've been better as `InferenceAggregationInferenceConfigContainer` which would be a breaking change but allows using `InferenceConfigContainer` within `ml/_types/inference`.